core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -294,6 +295,12 @@ object TopicCommand extends Logging {
             val partitionDesc = PartitionDescription(topicName, partition, Some(config), markedForDeletion = false)
             describeOptions.maybePrintPartitionDescription(partitionDesc)
           }
+        } catch {
+          case e: ExecutionException => {
+            print("\tTopic: " + topicName)
+            print("\t(LISTENER_NOT_FOUND)")
```

Comment:
What aboue other exceptions ? Isn't it we should make sure we capture each type of exceptions and handle them like leader not available, unknown topic or partition ?